### PR TITLE
Updates and clarifications

### PR DESCRIPTION
**Pull request by pirgeo**
Add a pointer to look at Spring Boot docs when using Micrometer with Spring boot, as in this case no DynatraceMeterRegistry should be instantiated.

### CODE REVIEW

File: src/docs/implementations/dynatrace.adoc [hunk 1]
```diff
@@ -242,9 +207,50 @@ DynatraceConfig dynatraceConfig = new DynatraceConfig() {
...
+        use-dynatrace-summary-instruments: true
+
+    # The export interval in which metrics are sent to Dynatrace (default: 60s).
+    step: 60s
+----
+
 For more information about the metadata picked up by the Dynatrace metadata enrichment feature, see https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/enrich-metrics/[the Dynatrace documentation].
 
-In Micrometer 1.9.0, Dynatrace-specific summary instruments (`DynatraceTimer` and `DynatraceDistributionSummary`) were introduced. These specialized instruments are tailored to the Dynatrace metrics ingest, and prevent the creation of invalid metrics. They are available from version 1.9.0 and are used as a drop-in replacement by default. No action is needed from users upgrading to 1.9.0. If there is a discrepancy in the observed metrics, it is possible to return to the previous behavior by setting the `useDynatraceSummaryInstruments` toggle to `false`.
+In Micrometer 1.9.0, Dynatrace-specific summary instruments (`DynatraceTimer` and `DynatraceDistributionSummary`) were introduced.
```
Comment (mviitane) [inline src/docs/implementations/dynatrace.adoc:250]: I wonder if it's agreed (if there's an official way) to manually create new lines or keep the whole paragraph in a single line in the source code. Now it's a bit messy with both methods being used in this document.

Comment (pirgeo) [reply src/docs/implementations/dynatrace.adoc:250]: Not sure if anything is enforced (I didn't find anything), but I think that each sentence is on its own line now everywhere in the file. I am used to this format now, and it doesn't make a difference in the rendered version.

**Review by mviitane — APPROVED** [review]
LGTM

File: src/docs/implementations/dynatrace.adoc [hunk 2]
```diff
@@ -57,7 +58,9 @@ MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTE
 
 `DynatraceConfig` is an interface with a set of default methods.
 Spring Boot's Micrometer support binds properties prefixed with `management.metrics.export.dynatrace` directly to the `DynatraceConfig`.
-This allows configuring the Dynatrace exporter by using the <<bookmark-available-properties, the available properties>>.
+This allows configuring the Dynatrace exporter by using <<bookmark-available-properties, the available properties>>.
+When using Micrometer in Spring Boot, you don't have to instantiate the `DynatraceMeterRegistry` manually, Spring Boot will do it automatically.
+All properties that can be set by overwriting methods can also be set via Spring Boot, and adding a separate MeterRegistry can lead to metrics not being exported.
```
Comment (arminru) [inline src/docs/implementations/dynatrace.adoc]:
```suggestion
All properties that can be set by overwriting methods can also be set via Spring Boot, and adding a separate MeterRegistry can lead to metrics not being exported as auto-configuration might break.
```

**Review by arminru — APPROVED** [review]

**Pull request merged**